catalog/core/catalog-core-api/src/main/java/ddf/catalog/plugin/PolicyResponse.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.plugin;
+
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Response from PolicyPlugin containing policy information for {@link ddf.catalog.operation.Operation}
+ * and individual items, such as {@link ddf.catalog.data.Metacard} or {@link ddf.catalog.data.Result}
+ */
+public interface PolicyResponse {
+
+    /**
+     * Policy to be applied to the catalog operation being attempted.
+     *
+     * @return Map containing policy information pertaining to this operation.
```

Comment:
Can we provide more details on the keys and values used in the Maps and Sets? For instance, what they represent and how they are used. Or maybe refer to some JavaDoc that explains how policies are used and enforced? This would be really useful for whoever implements plugins or uses the policies.